### PR TITLE
Update DCO

### PR DESCRIPTION
**Pull request by hoseinipeyrov**
Please check the following:

1. Make sure you are targeting the `master` branch, pull requests on release branches are only allowed for bug fixes.
2. Read contributing guidelines: https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md
3. Describe what your pull request does and which issue you're targeting (if any)

**You MUST delete the content above including this line before posting, otherwise your pull request will be invalid.**

### CODE REVIEW

File: DCO [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Developer Certificate of Origin
 Version 1.1
 
-Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+Copyright (C) 2004, 20019 The Linux Foundation and its contributors.
```
Comment (techknowlogick) [inline DCO:4]:
```suggestion
Copyright (C) 2004, 2019 The Linux Foundation and its contributors.
```

**Review by techknowlogick — CHANGES_REQUESTED** [review]
I'm unsure of this change, because the official document at https://developercertificate.org/ still has 2006

File: DCO (hunk 1 above)
Comment (gary-kim) [inline DCO:4]: This should still stay as 2006. Copyright years only get updated if the copyrighted material itself is updated or modified.

**Review by gary-kim — CHANGES_REQUESTED** [review]

**Comment by techknowlogick** [comment]
Thank you for your PR. I will be closing this as per above comments.

**Pull request closed without merging**